Wikipedia/Localizations/qqq.lproj/Localizable.strings

```diff
@@ -334,14 +335,17 @@
 "settings-title" = "Title of the view where app settings are displayed.\n{{Identical|Settings}}";
 "share-a-fact-share-menu-item" = "Button label for creating a Share-a-fact card from the current text selection";
 "share-article-name-on-wikipedia" = "Formatted string expressing article being on Wikipedia with at symbol handle. Please do not translate the \"@Wikipedia\" in the message, and preserve the spaces around it, as it refers specifically to the Wikipedia Twitter account. $1 will be an article title, which should be wrapped in the localized double quote marks.";
-"share-article-name-on-wikipedia-with-selected-text" = "Formatted string expressing article being on Wikipedia with at symbol handle, with a user-selected snippet. Please do not translate the \"@Wikipedia\" in the message, and preserve the spaces around it, as it refers specifically to the Wikipedia Twitter account. $1 will be an article title, which should be wrapped in the localized double quote marks. $2 will be a user-selected text snippet, which should be wrapped in the localized double quote marks.";
-"share-as-image" = "Button label for sharing as an image card";
-"share-as-text" = "Button label for sharing as a text snippet (instead of as an image card)";
-"share-cancel" = "Button which dismisses the share dialog, cancelling the action.\n{{Identical|Cancel}}";
+"share-building" = "Shown while Share-a-fact card is being constructed";
 "share-custom-menu-item" = "Button label for text selection Share\n{{Identical|Share}}";
+"share-default-format" = "Default share format string. $1 is replaced with the selected text, $2 is replaced with the article title, $3 is replaced with the articleURL.";
```

Comment:
Consider changing "share format string" here to match all of the others, which use "share string format".